support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -606,6 +614,19 @@ export function CheckoutComponent({
 			}
 		}
 
+		if (paymentMethod === 'DirectDebit') {
+			const newDirectDebitFieldError = {
+				...(!recaptchaToken && { recaptcha: 'Please complete security check' }),
+			};
+
+			// Don't go any further if there are errors for any Direct Debit fields
+			if (Object.values(newDirectDebitFieldError).some((value) => value)) {
```

Comment:
Do you think we could simply this to 
```suggestion
			if (newDirectDebitFieldError.recaptcha) {
```

Reply:
Addressed in 4f38659